src/pages/trader-tools/common/_style.js

```diff
@@ -104,23 +117,57 @@ export const SwapFormWrapper = styled(FormWrapper)`
     }
 `
 
-export const CalculatorForm = styled(Form)`
+const CalculatorFormStyles = css`
     background-color: var(--color-white);
     border-radius: 10px;
     box-sizing: border-box;
     box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.05), 0 0 20px 0 rgba(0, 0, 0, 0.05);
-    width: 48.6rem;
 
     @media ${device.mobileL} {
-        width: 328px;
         margin-bottom: 20px;
+        width: 328px;
     }
 `
 
+export const CalculatorForm = styled(Form)`
+    ${CalculatorFormStyles}
+    width: 54rem;
+`
+export const PnlCalculatorFormMobile = styled(Form)`
+    ${CalculatorFormStyles}
+    width: 328px;
+`
+
+export const PnlHeaderOverflow = styled.div`
+    width: 328px;
+    border-radius: 8px 8px 0 0;
+    overflow-x: scroll;
+`
+const CalculatorHeaderStyles = css`
+    border-radius: 8px 8px 0 0;
+    padding: 2.4rem;
+    background-color: var(--color-blue-4);
+`
+
 export const CalculatorHeader = styled.div`
+    ${CalculatorHeaderStyles}
+`
+
+export const PnlCalculatorHeader = styled.div`
+    ${CalculatorHeaderStyles}
+
+    @media (max-width: 424px) {
+        width: 440px;
```

Comment:
I think we need to handle the mobile screen until 375px
currently , we need to readjust the styling for it (i.e. smaller text and width)

Reply:
Oh this was done mandatory according to the design. It's a scrollable section hence it will be larger than the viewport.